src/pages/Overview/OverviewPage.tsx

```diff
@@ -376,12 +381,18 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
   fetchValidationChunk(chunk: NamespaceInfo[]) {
     return Promise.all(
       chunk.map(nsInfo => {
-        return API.getNamespaceValidations(nsInfo.name).then(rs => ({ validations: rs.data, nsInfo: nsInfo }));
+        return Promise.all([
+          API.getNamespaceValidations(nsInfo.name),
+          API.getIstioConfig(nsInfo.name, ['authorizationpolicies'], false, '', '')
```

Comment:
Should we add the `kiali_wizard=AuthorizationPolicy` in this call? This way, clicking into the `Delete Authz Policies` will only delete the AuthzPolicies created by this wizard.

Reply:
I was unsure but now you bring this I guess we should keep consistency here and only delete the "managed" resources and let user delete other assets.

Reply:
It seems a bit of a toggle action: create a set of policies, remove that set of policies. Removing other policies would lead to some frustration, specially because there isn't any alert saying what is happening after clicking this. I can imagine people in production performing this operation and removing extra policies.

Reply:
Actually we delete all info, not only the Kiali generated one, so I vote to keep this as it is at least in a first iteration.